vizro-core/src/vizro/static/css/layout.css

```diff
@@ -233,3 +233,35 @@
   height: 100%;
   overflow: auto;
 }
+
+.container-title {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* Change cursor for the whole container title when it's a collapsible container.
+Doing has(.material-symbols-outlined) isn't a very nice way to identify when the title
+is for a collapsible container but due to the order of elements, sibling selectors
+with collapsible-container don't work here. */
```

Comment:
@huong-li-nguyen can you see any nicer way to do this?

Also should we move all this stuff to a new `container.css` file?

Reply:
Cool, I was thinking of doing it that way too but didn't know if we wanted the two different `container-title-collapse` vs. `container-title` classes. It's definitely more robust than the current solution though so let's do it 👍